code/game/gamemodes/cult/talisman.dm

```diff
@@ -1,5 +1,6 @@
 /obj/item/weapon/paper/talisman
 	icon_state = "paper_talisman"
+	origin_tech = list(TECH_ARCANE = 4)

```

Comment:
This is similar to a tome, though since it only makes use of one "arcane" law, and is relatively common in cult rounds, it's likely a 3,